src/jp/news/blog/search-output.html

```diff
@@ -4,6 +4,6 @@
 override:tags: []
 ---
 
-[ {% for item in collections['en-GB-blog-post'] %} { "image":"{{item.data.image}}", "title":"{{item.data.title}}",
```

Comment:
Ah.. or maybe github got confused about which files to rename

Reply:
Yeah, good spot @djgalloway. That caught my eye too. Ran the build locally and all was looking good. I think GitHub got muddled with the rename

Reply:
Out of pure curiosity, @Pete-Robelou, did you use `mv` or `git mv` locally?

Reply:
I didn't use either. I manually updated all file/folder names and any syntax within them.

Everything is as it should be on my local machine but like you say, I guess GitHub mixed up the `news/blog/search-output.html` files.

Am I correct in saying that it still works as the `en-GB/.../search-output.html` was renamed to `jp/.../search-output.html` and `ja-JP/.../search-output.html` was renamed to `en/.../search-output.html` ?

Reply:
Yeah I'm sure it still works fine.  Just odd.